NYTPhotoViewer/NYTInterstitialViewController.m

```diff
@@ -14,6 +14,7 @@ @interface NYTInterstitialViewController ()
 @property (nonatomic, nullable) id <NYTPhoto> photo;
 @property (nonatomic, nullable) UIView *interstitialView;
 @property (nonatomic) NSUInteger photoViewItemIndex;
+@property (nonatomic, nullable) NSArray<NSLayoutConstraint *> *constraints;
```

Comment:
nit: generally good practice to defensively use `copy` for properties of types with mutable subclasses to prevent unexpected mutation out from under you. Not an issue currently, but that’s why it’s defensive.